chips/sam4l/src/trng.rs

```diff
@@ -25,7 +47,7 @@ pub struct Trng<'a> {
 }
 
 pub static mut TRNG: Trng<'static> = Trng::new();
-const KEY: u32 = 0x524e4700;
+const KEY: u32 = 0x524e47;
```

Comment:
Should this have changed?

Reply:
Yes, the 8 least significant zeros were padding so that it didn't have to be shifted each time.